draft-ietf-quic-recovery.md

```diff
@@ -274,36 +271,35 @@ alarm for TCP as well, and this document incorporates this advancement.
 
 ## Timer-based Detection
 
-Timer-based loss detection implements a handshake retransmission timer that
-is optimized for QUIC as well as the spirit of TCP's Tail Loss Probe
-and Retransmission Timeout mechanisms.
+Timer-based loss detection implements a handshake retransmission timer that is
+optimized for QUIC as well as the spirit of TCP's Tail Loss Probe and
+Retransmission Timeout mechanisms.
 
 ### Crypto Handshake Timeout
 
-Data in CRYPTO_HS frames is critical to QUIC transport and crypto negotiation,
-so a more aggressive timeout is used to retransmit it.  Below, the word
-handshake packet is used to refer to packets containing CRYPTO_HS frames,
-not packets with the specific long header packet type Handshake.
+Data in CRYPTO frames is critical to QUIC transport and crypto negotiation, so a
+more aggressive timeout is used to retransmit it.  Below, the word handshake
+packet is used to refer to packets containing CRYPTO frames, not packets with
```

Comment:
(Unmodified text, but you touch it, you own it.)  "handshake packet" isn't actually *a* word.  Perhaps "the term," and put "handshake packet" in quotes to designate it as the term you're discussing for clarity?